CONTRIBUTING.md

```diff
@@ -1,269 +1,70 @@
+cd exonum
+cargo test --all
 ```
 
-Please refrain from creating several pull requests for the same change.
-Use the pull request that is already open (or was created earlier) to amend changes.
-This preserves the discussion and review that happened earlier
-for the respective change set.
-
-The length of time required for peer review is unpredictable and will vary
-from pull request to pull request.
-
-## Pull Request Philosophy
-
-Patchsets should always be focused. For example, a pull request could add a feature,
-fix a bug, or refactor code; but not a mixture.
-Please also avoid super pull requests which attempt to do too much,
-are overly large, or overly complex as this makes review difficult.
-
-### Features
-
-When adding a new feature, thought must be given to the long term technical debt
-and maintenance that feature may require after inclusion.
-Before proposing a new feature that will require maintenance,
-please consider if you are willing to maintain it (including bug fixing).
-If features get orphaned with no maintainer in the future, they may be removed
-by the Repository Maintainer.
-
-### Refactoring
-
-Refactoring is a necessary part of any software project's evolution.
-The following guidelines cover refactoring pull requests for the project.
-
-There are three categories of refactoring, code only moves,
-code style fixes, code refactoring.
-In general refactoring pull requests should not mix these three kinds
-of activity in order to make refactoring pull requests easy to
-review and uncontroversial. In all cases, refactoring PRs must not change
-the behaviour of code within the pull request (bugs must be preserved as is).
-
-Project maintainers aim for a quick turnaround on refactoring pull requests,
-so where possible keep them short, uncomplex and easy to verify.
-
-## "Decision Making" Process
-
-The following applies to code changes to the Exonum Core project and related
-projects such as Exonum BTC Anchoring Service.
-
-Whether a pull request is merged into Exonum Core rests with the project merge
-maintainers and ultimately the project lead.
-
-Maintainers will take into consideration if a patch is in line with the general
-principles of the project; meets the minimum standards for inclusion;
-and will judge the general consensus of contributors.
-
-In general, all pull requests must:
+## Finding something to fix or improve
 
-- have a clear use case, fix a demonstrable bug or serve the greater good
-    of the project (for example refactoring for modularisation);
-- be well peer reviewed;
-- have unit tests and sandbox tests where appropriate;
-- follow code style guidelines;
-- not break the existing test suite;
-- where bugs are fixed, where possible, there should be unit tests demonstrating
-    the bug and also proving the fix. This helps prevent regression.
+The [good first issue] label can be used to find the easy issues. If you want
+to work on an issue, please leave a comment so that we can assign it to you.
```

Comment:
I am mildly against 

>If you want to work on an issue, please leave a comment so that we can assign it to you.

clause, for two reasons:

1) It is only occasionally helpful, because clashes rarely happen in practice. 
2) It discourages contributors, because it asks for commitment to fix the issue. That is, it "forbids" the following workflow: "I'll want to fix an issue, but I am not sure about my abilities to do it, so I'll just *try* to fix it locally and, if I am successful, I'll send a PR. If, however, I fail to fix the bug, nobody knows that about that!"

It might be a good idea for *ourselves* though to mark formerly "help wanted" issues if we decide to fix them ourselves.

Reply:
Removed. :+1: